ports/atmel-samd/boards/circuitplayground_express_crickit/board.c

```diff
@@ -53,5 +53,5 @@ bool board_requests_safe_mode(void) {
 }
 
 void reset_board(void) {
-    board_reset_user_neopixels();
+    board_reset_user_neopixels(&pin_PB23, 10);
```

Comment:
The number of neopixels could be a `#define` also.

Reply:
Since it's only used here, I'd rather have it be inline. If we need it in more places, then we can factor it out. (Same with the pin define.)